troubleshooting/debugging.md

```diff
@@ -98,6 +99,43 @@ Even if you see this message, you still have not lost logs yet. Since it will re
 
 When you see this message, you have lost logs. The other case that indicates log loss is when an input is paused, which is covered in the [overlimit error section](#overlimit-warnings).
 
+#### Common Network Errors
+
+First, please read the section [Network Connection Issues](#network-connection-issues), which explains that Fluent Bit's http client is very low level and logs errors that other clients may simply silently retry. That section also explains the `auto_retry_requests` option which now defaults to true. 
+
+Your Fluent Bit deployment will likely occasionally log network connection errors. These are normal and if they occur only occasionally and do not lead log loss, should not be a cause for concern. See [How do I tell if Fluent Bit is losing logs?](#how-do-i-tell-if-fluent-bit-is-losing-logs) to learn how to check for log loss from network issues. If you see network errors happen very frequently or they cause your retries to expire, then please cut us an issue.
```

Comment:
[nit-pick]
- likely occasionally -> occasionally
- These are normal and if they occur only occasionally and do not lead log loss, should not be a cause for concern - These are normal and should not cause concern unless...